python/kserve/test/test_v1beta1_external_metrics.py

```diff
@@ -15,12 +15,12 @@
 # coding: utf-8
 
 """
-    KServe
```

Comment:
These files are auto-generated, were these changes result of the `make precommit` command?